app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
Sorry, i hadn't caught this -- I believe there are maps with the same name but different dates. This is for when people re-map the same site. So I believe we need the friendly_id of maps to include the date.